layouts/_default/li.html

```diff
@@ -1,6 +1,6 @@
 <li>
-    <a href="{{ .Permalink }}">
-        {{ .Title }}
-        <small><time>{{ .Date.Format "Jan 2, 2006" }}</time></small>
-    </a>
+  <a href="{{ .Permalink }}">
+    {{ .Title }}
+    <small><time>{{ .Date | time.Format (i18n "posts.date") }}</time></small>
+  </a>
```

Comment:
Use four spaces here instead of two, to remain consistent with the rest of the codebase.